sdk/eventhub/event-hubs/vitest.browser.config.ts

```diff
@@ -44,5 +45,10 @@ export default defineConfig({
       reporter: ["text", "json", "html"],
       reportsDirectory: "coverage-browser",
     },
+    alias: {
+      "@azure/event-hubs": resolve("./dist/browser/index.js"),
+      "../../../src": resolve("./dist/browser"),
+      "../../src": resolve("./dist/browser"),
```

Comment:
Why two different paths for same `./dist/browser`?

Reply:
we want all imports from the top level to be from the production browser build